### PR TITLE
No line breaks in severe log alerts #4896

### LINKED ISSUE #4896: No line breaks in severe log alerts
This problem doesn't seem to be fixed yet (this is from V5.70):
image

### PR DESCRIPTION
**Pull request by wkurniawan07**
Fixes #4896

### CODE REVIEW

File: src/main/java/teammates/logic/core/Emails.java [hunk 1]
```diff
@@ -806,7 +806,7 @@ public MimeMessage sendErrorReport(HttpServletRequest req, Throwable error) {
             log.severe("Sent crash report: " + Emails.getEmailInfo(email));
         } catch (Exception e) {
             log.severe("Crash report failed to send. Detailed error stack trace: "
-                     + TeammatesException.toStringWithStackTrace(error));
+                     + TeammatesException.toStringWithStackTrace(error).replace("\\n","<br>"));
```
Comment (damithc) [inline src/main/java/teammates/logic/core/Emails.java]: Can this be inserted into the `toStringWithStackTrace()` itslef?

**Comment by thyageshm** [comment]
@wkurniawan07 I believe the reason why the emails are not being converted is because I used this:
```
.replace("\\n", "<br>")
```
instead of this:
```
.replace("\\n", "<br>")
```

So I think the fix for this should be in the Emails.java line 641

**Comment by wkurniawan07** [comment]
@thyageshm changes made. Somehow I had a feeling that some cases might still run loose, though.

File: src/main/java/teammates/logic/core/Emails.java [hunk 2]
```diff
@@ -824,7 +824,7 @@ public MimeMessage sendLogReport(MimeMessage message) {
     
     private void logSevereForErrorInSendingItem(String itemType, MimeMessage message, Exception e) {
         log.severe("Error in sending " + itemType + ": " + (message == null ? "" : message.toString())
-                   + "\nCause: " + TeammatesException.toStringWithStackTrace(e));
```
Comment (thyageshm) [inline src/main/java/teammates/logic/core/Emails.java]: I think we should remove the white space change

**Comment by thyageshm** [comment]
I think the logs themselves still have the `\n` as line break, but since we are changing the entire body of the email, it *should* cover all cases. Unless we have other means of sending emails....

File: src/main/java/teammates/logic/core/Emails.java (hunk 2 above)
Comment (wkurniawan07) [inline src/main/java/teammates/logic/core/Emails.java]: Done!

**Comment by thyageshm** [comment]
Good to go! Please help me add the tomerge label. Thanks!

**Comment by thyageshm** [comment]
@wkurniawan07 please merge this now. I am going make this release very soon

**Comment by wkurniawan07** [comment]
@thyageshm dude you asked to merge now and then you went on and merged #4875, now I have to repeat merging all over again...
I hope no one else is in the merge queue now :p

**Comment by thyageshm** [comment]
I wasn't sure when exactly you would do it... haha sorry about that.

**Comment by thyageshm** [comment]
So can you merge this? Its getting late and I want close this release.

**Comment by wkurniawan07** [comment]
I'm running tests right now. Should be done in about 5 mins.

**Comment by thyageshm** [comment]
Great! thanks :-) Would you be able to merge the other two as well? or shall we push them to the next release? I can wait if you want to do them now as well.

**Comment by wkurniawan07** [comment]
There are 4 ready-for-merge PRs actually. I can merge them all now if needed, are you willing to wait?

**Comment by thyageshm** [comment]
Yes

**Comment by wkurniawan07** [comment]
@thyageshm done. I can't merge the last one because of conflicts.

**Comment by thyageshm** [comment]
Awesome, thanks!

**Pull request merged**